### PR TITLE
Added some documentation

### PR DESCRIPTION
**Pull request by serhii-petrishenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,82 @@
...
+```
+
+### Set timeout interval, in seconds
+```swift
+TLManager.shared.setTimeoutInterval(10)
+```
+
+### Set environment
+```swift
+TLManager.shared.setEnvironment(.production)
```
Comment (BMacLinden) [inline README.md:31]: let's default to .staging

File: README.md [hunk 2]
```diff
@@ -1,2 +1,82 @@
-# tilia-ios-sdk
-Tilia iOS SDK
+# Tilia iOS SDK
+
+This is a repository for a Tilia iOS SDK. It includes demo app that shows all possibilities and flows.
```
Comment (BMacLinden) [inline README.md:3]: recommend changing to: 
"""
This is the repository for the Tilia iOS SDK.
This SDK is meant to be used in concert with your own api server which communicates with Tilia APIs to retrieve authorized user access tokens and any additional data required by the SDK flow you are looking to use.
This repository includes an interactive demo application which can guide you through what data is expected for each flow as well as how to customize the look of the SDK UI.
"""

**Review by BMacLinden — APPROVED** [review]
Added a couple of recommended changes

File: README.md (hunk 2 above)
Comment (serhii-petrishenko) [reply README.md]: done

**Pull request merged**